Source/WebCore/accessibility/AccessibilityObject.cpp

```diff
@@ -671,6 +671,11 @@ void AccessibilityObject::insertChild(AccessibilityObject& child, unsigned index
     }
 #endif // USE(ATSPI)
 
+    auto insert = [this] (Ref<AXCoreObject>&& object, unsigned index) {
+        setChildIndexInParent(object.get(), index);
```

Comment:
Is this one ignoring the return value? As mentioned in the other comment, maybe it should just be a debug check? Or alternatively maybe use std::ignore to make it explicit that you're ignoring the return value?

Reply:
Yeah, in this case we intentionally ignore the return value, since the insertion index is managed elsewhere in this function. But `std::ignore` would be good to make it explicit, I'll do that.